engines/director/resource.cpp

```diff
@@ -251,24 +229,33 @@ Archive *Window::loadEXE(const Common::String movie) {
 	} else {
 		warning("No LINGO.INI");
 	}
+}
 
+Archive *Window::loadEXE(const Common::String movie) {
 	Common::SeekableReadStream *exeStream = SearchMan.createReadStreamForMember(Common::Path(movie, g_director->_dirSeparator));
 	if (!exeStream)
-		return nullptr;
+		error("Failed to open EXE '%s'", g_director->getEXEName().c_str());
```

Comment:
Please put the method name for easier debugging, e.g. `Window::loadEXE(): Failed to ope...` and the same with the error messages below.

Reply:
Done